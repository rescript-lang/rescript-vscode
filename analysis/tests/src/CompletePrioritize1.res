module Test = {
  type t = {name: int}
  let add = (a: float) => a +. 1.0
}
let a: Test.t = {name: 4}
// a->
//    ^com
