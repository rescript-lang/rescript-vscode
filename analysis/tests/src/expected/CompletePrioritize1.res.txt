Complete src/CompletePrioritize1.res 5:6
posCursor:[5:6] posNoWhite:[5:5] Found expr:[5:3->0:-1]
Completable: Cpath Value[a]->
ContextPath Value[a]->
ContextPath Value[a]
Path a
CPPipe env:CompletePrioritize1
CPPipe type path:Test.t
Path Test.
[{
    "label": "Test.add",
    "kind": 12,
    "tags": [],
    "detail": "float => float",
    "documentation": null
  }]

